flake.nix

```diff
@@ -23,120 +23,27 @@
     };
   };
 
-  outputs = { self, nixpkgs, ... }@inputs:
+  outputs =
+    { nixpkgs, ... }@inputs:
     let
-      supportedSystems = nixpkgs.lib.systems.flakeExposed;
-
-      pkgsFor = nixpkgs.lib.genAttrs supportedSystems (system: rec {
-        pkgs = import nixpkgs {
-          inherit system;
-          overlays = [
-            self.overlays.wasm-bindgen-cli
-            inputs.rust-overlay.overlays.default
-          ];
-        };
-
-        topiaryPkgs = pkgs.callPackage ./default.nix {
+      forAllSystems = nixpkgs.lib.genAttrs nixpkgs.lib.systems.flakeExposed;
+
+      nix = forAllSystems (
```

Comment:
Nitpick: it's a bit confusing to call that variable `nix` (as we are already in a Nix file). I was confused, as I started to look at the new output, as my brain read it as the `nix` package from `pkg.nix` instead. Maybe something like `topiary` or `topiary-packages` or whatnot would be more specific?

Reply:
Yeah, fair. I hesitated calling it `this`. I'm not super convinced by `topiary` or `topiary-packages`, seeing as it contains more stuff, but that is probably still better than `nix`. :+1: